.github/ISSUE_TEMPLATE/broken_addon.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Report a broken add-on
+about: Report an add-on that is currently broken within Kodi.  
+title: ''
+labels: 'Broken-Addon'
+assignees: ''
+
+---
+
+
+### Add-on details:
```

Comment:
maybe add a checkbox where the requester can select whether he is the author of the addon? We need to check if either way, so feel free to ignore.

Reply:
Good point. I added a check-box.